src/multimeasurerest.ts

```diff
@@ -125,14 +119,14 @@ export class MultiMeasureRest extends Element {
   }
 
   drawLine(ctx: RenderContext, left: number, right: number, sbl: number): void {
-    const y = this.checkStave().getYForLine(this.render_options.line);
+    const y = this.checkStave().getYForLine(this.render_options.line as number);
     const padding = (right - left) * 0.1;
 
     left += padding;
     right -= padding;
 
     const serif = {
-      thickness: this.render_options.serif_thickness,
+      thickness: this.render_options.serif_thickness as number,
```

Comment:
Ahh, this one is tough! Even in the final product, I did not use Required.

That's because MultiMeasureRest supports two optional fields (padding_left and padding_right). These fields are calculated on the fly in the draw() method. So if I want to use Required, I might have to create a whole separate interface for this. 

See: 
https://github.com/ronyeh/vexflow/blob/3118234a440b0549e6d89fea09a145861a70a621/src/multimeasurerest.ts#L91-L102

Let me know if you have a better solution for this one! It stumped me, for sure.